apps/web/pages/[user].tsx

```diff
@@ -179,50 +239,53 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   const web3Credentials = credentials.find((credential) => credential.type.includes("_web3"));
 
-  const eventTypesWithHidden = await prisma.eventType.findMany({
-    where: {
-      AND: [
-        {
-          teamId: null,
-        },
-        {
-          OR: [
+  const eventTypesWithHidden =
```

Comment:
Probably moving to a private function so it gains some readability.
`private getEventTypesWithHiddenFromDB()`

`const eventTypesWithHidden = usernameList.length > 1 ? [] : await getEventTypesWithHiddenFromDB()`